Modules/Microsoft365DSC/DSCResources/MSFT_IntuneMobileAppsWindowsOfficeSuiteApp/MSFT_IntuneMobileAppsWindowsOfficeSuiteApp.schema.mof

```diff
@@ -0,0 +1,83 @@
+    [Write, Description("Specifies whether to exclude Microsoft Office InfoPath from the installation.")] Boolean InfoPath;
+    [Write, Description("Specifies whether to exclude Microsoft Office Skype for Business (Lync) from the installation.")] Boolean Lync;
+    [Write, Description("Specifies whether to exclude Microsoft Office OneDrive from the installation.")] Boolean OneDrive;
+    [Write, Description("Specifies whether to exclude Microsoft Office OneNote from the installation.")] Boolean OneNote;
+    [Write, Description("Specifies whether to exclude Microsoft Office Outlook from the installation.")] Boolean Outlook;
+    [Write, Description("Specifies whether to exclude Microsoft Office PowerPoint from the installation.")] Boolean PowerPoint;
+    [Write, Description("Specifies whether to exclude Microsoft Office Publisher from the installation.")] Boolean Publisher;
+    [Write, Description("Specifies whether to exclude Microsoft Office SharePoint Designer from the installation.")] Boolean SharePointDesigner;
+    [Write, Description("Specifies whether to exclude Microsoft Office Teams from the installation.")] Boolean Teams;
+    [Write, Description("Specifies whether to exclude Microsoft Office Visio from the installation.")] Boolean Visio;
+    [Write, Description("Specifies whether to exclude Microsoft Office Word from the installation.")] Boolean Word;
+};
+
+[ClassVersion("1.0.0"), FriendlyName("IntuneMobileAppsWindowsOfficeSuiteApp")]
+class MSFT_IntuneMobileAppsWindowsOfficeSuiteApp : OMI_BaseResource
+{
+    [Key, Description("The admin provided or imported title of the app. Inherited from mobileApp.")] String DisplayName;
+    [Write, Description("The unique identifier for an entity. Read-only. Inherited from mobileApp object.")] String Id;
+
+    [Write, Description("The description of the app. Inherited from mobileApp.")] String Description;
+    [Write, Description("The publisher of the app. Inherited from mobileApp.")] String Publisher;
+    [Write, Description("The value indicating whether the app is marked as featured by the admin. Inherited from mobileApp.")] Boolean IsFeatured;
+    [Write, Description("The privacy statement Url. Inherited from mobileApp.")] String PrivacyInformationUrl;
+    [Write, Description("The InformationUrl of the app. Inherited from mobileApp.")] String InformationUrl;
+    [Write, Description("The owner of the app. Inherited from mobileApp.")] String Owner;
+    [Write, Description("The dewveloper of the app. Inherited from mobileApp.")] String Developer;
+    [Write, Description("Notes for the app. Inherited from mobileApp.")] String Notes;
+    [Write, Description("The publishing state for the app. The app cannot be assigned unless the app is published. Inherited from mobileApp."), ValueMap{"notPublished", "processing","published"}, Values{"notPublished", "processing", "published"}] String PublishingState;
```

Comment:
That property is read-only and can't be changed. Should be removed.

Reply:
Have removed them, thank you!